source/style-guide/markup/directives/metadata.txt

```diff
@@ -1,20 +1,28 @@
+.. _metadata:
+
 ========================
 Page Metadata Directives
 ========================
 
 meta
 ----
 
-Generates HTML ``<meta>`` tags.
+Use ``.. meta::`` to add |html| meta tags to a page. Adding meta tags 
+aids in searches both internally and via the Internet. You can add both 
```

Comment:
[issue]

Our style guide suggests avoiding "via". Maybe consider "through the Internet" instead?

Style guide link: https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#v